.github/workflows/build_and_test.yml

```diff
@@ -884,7 +884,7 @@ jobs:
 
       - name: Push images to production ECR
         if: |
-          (github.ref_name == 'main' || github.ref_name == 'release') &&
+          github.ref_name == 'main' &&
            github.event_name != 'workflow_dispatch'
```

Comment:
Probably this part we should do only for `release` branch, what do you think guys @zoete @SergeyMelnikov?

Reply:
Well, if we do this we cannot catch integration issues early (because we use latest in CI). Also this leads to a longer time waiting for other teams to start the work that depends on a neon feature. Let's have another 'trunk' tag for current main then?

Reply:
Yes, it's docker. Sorry, I assumed it was a generic comment and not to a single step